packages/hardware-trezor/test/transformers/withdrawals.test.ts

```diff
@@ -0,0 +1,64 @@
+import { Cardano } from '@cardano-sdk/core';
+import { CardanoKeyConst, util } from '@cardano-sdk/key-management';
+import {
+  contextWithKnownAddresses,
+  contextWithoutKnownAddresses,
+  coreWithdrawalWithKeyHashCredential,
+  coreWithdrawalWithScriptHashCredential,
+  stakeKeyHash,
+  stakeScriptHash
+} from '../testData';
+import { mapWithdrawals, toWithdrawal } from '../../src/transformers';
+
+describe('withdrawals', () => {
+  describe('mapWithdrawals', () => {
+    it('return undefined if given an undefined object as withdrawal', async () => {
+      const withdrawals: Cardano.Withdrawal[] | undefined = undefined;
```

Comment:
Can this happen in real scenarios? And if yes, why wouldn't we provide an empty array?

Reply:
Because Trezor is not expecting withdrawals param if there are no withdrawals

Reply:
changed!